README.md

```diff
@@ -76,6 +76,17 @@ Check out the [documentation][20] for pages on a variety of topics, and see the
 
 For Glide v3, see the [wiki][2].
 
+Before Implementaion, you will need to add the following storage permissions to your application's manifest. For Android SDK version 23(M) and above, you will also need to explicitly request these permissions from the user.
```

Comment:
This is already listed on the documentation which is linked immediately above this: http://bumptech.github.io/glide/doc/download-setup.html#permissions